.github/workflows/go-tests.yaml

```diff
@@ -35,12 +35,7 @@ jobs:
 
       - name: install libyara-dev
         run: |
-          sudo add-apt-repository -n -y "deb http://archive.ubuntu.com/ubuntu/ mantic main restricted universe multiverse"
-          sudo add-apt-repository -n -y "deb http://archive.ubuntu.com/ubuntu/ mantic-updates main restricted universe multiverse"
-          sudo add-apt-repository -n -y "deb http://archive.ubuntu.com/ubuntu/ mantic-backports main restricted universe multiverse"
-          sudo add-apt-repository -n -y "deb http://security.ubuntu.com/ubuntu mantic-security main restricted universe multiverse"
-
-          sudo apt update && sudo apt install libyara-dev -y
+          make build-yara
```

Comment:
This looks to be as fast or faster than running `sudo apt update && sudo apt install libyara-dev -y`.